Some Editorial changes (#9)
ietf-wg-nmop/Logistic: IAB Workshop.md

Those requirements were instrumental in developing first the NETCONF
protocol (in the NETCONF Working Group) [RFC6241], the associated YANG
data modeling language (in the NETMOD Working Group) [RFC7950], 

> RESTCONF [RFC8040], and most recently CORECONF [I.D-ietf-core-comi].

Reviewer: ```suggestion
RESTCONF [RFC8040], and most recently CORECONF [I-D.ietf-core-comi].
```

# Laying a path for the future of Network Management

Network topologies have become significantly more complex since the
2002 IAB workshop, as have the router and host technologies. 

> Assuming this trend will continue, this workshop's incentive is

Reviewer: ```suggestion
Assuming this trend will continue, this workshop's ambition is
```

to lay a new directional foundation for the continued
future of network management protocols, gathering new input on 
what new issues network operators and network management implementers are facing.

The candidate items for discussion include, but are not limited to:

- Tooling, opensource, experimentation, proof of concept, multi-vendor interoperability test (e.g., EANTC),
  system integration

> - Data consistency to support richer full stack observability (Data & Knowledge)

Reviewer: ```suggestion
- Data consistency to support richer observability (Data & Knowledge)
```

Not sure "full stack" will understood. I'm having troubles myself with parsing its meaning here.

# Out of scope topics

A workshop dedicated to management techniques related to encrypted

> networks has recently been held [RFC9490]; Another workshop dedicated

Reviewer: ```suggestion
networks has recently been held [RFC9490]. Another workshop dedicated
```

> to environment impact of internet application and system has also been held 

Reviewer: ```suggestion
to environment impact of Internet applications and systems has also been held 
```

> [RFC9547]. These topics are therefore out of scope for this workshop.

Reviewer: ```suggestion
[RFC9547]. These topics are, therefore, out of scope for this workshop.
```
